content/upload/1-upload.md

```diff
@@ -411,18 +331,352 @@ SELECT
+    OS_NAME,
+    OS_FAMILY,
+    OS_MANUFACTURER,
+    OS_TIMEZONE,
+    DVCE_TYPE,
+    DVCE_ISMOBILE,
+    DVCE_SCREENWIDTH,
+    DVCE_SCREENHEIGHT,
+    DOC_CHARSET,
+    DOC_WIDTH,
+    DOC_HEIGHT,
+    TR_CURRENCY,
+    TR_TOTAL_BASE,
+    TR_TAX_BASE,
+    TR_SHIPPING_BASE,
+    TI_CURRENCY,
+    TI_PRICE_BASE,
+    BASE_CURRENCY,
+    GEO_TIMEZONE,
+    MKT_CLICKID,
+    MKT_NETWORK,
+    ETL_TAGS,
+    DVCE_SENT_TSTAMP,
+    REFR_DOMAIN_USERID,
+    REFR_DVCE_TSTAMP,
+    DOMAIN_SESSIONID,
+    DERIVED_TSTAMP,
+    EVENT_VENDOR,
+    EVENT_NAME,
+    EVENT_FORMAT,
+    EVENT_VERSION,
+    EVENT_FINGERPRINT,
+    TRUE_TSTAMP,
+    STRUCT(ID, NAME, PREVIOUS_ID, PREVIOUS_NAME, PREVIOUS_TYPE, TRANSITION_TYPE, TYPE) AS UNSTRUCT_EVENT_COM_SNOWPLOWANALYTICS_MOBILE_SCREEN_VIEW_1,
+   FROM_JSON(CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_CLIENT_SESSION_1, 'ARRAY<STRUCT<SESSION_ID:STRING, SESSION_INDEX:STRING, PREVIOUS_SESSION_ID:STRING, USER_ID:STRING, FIRST_EVENT_ID:STRING, EVENT_INDEX:STRING, STORAGE_MECHANISM:STRING, FIRST_EVENT_TIMESTAMP:STRING>>') AS CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_CLIENT_SESSION,
+   FROM_JSON(CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_GEOLOCATION_CONTEXT_1, 'ARRAY<STRUCT<LATITUDE:FLOAT, LONGITUDE:FLOAT, LATITUDE_LONGITUDE_ACCURACY:FLOAT, ALTITUDE:FLOAT, ALTITUDE_ACCURACY:FLOAT, BEARING:FLOAT, SPEED:FLOAT, TIMESTAMP:INT>>') AS CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_GEOLOCATION_CONTEXT
+
+
+  FROM STRUCT_BASE
+
+)
 ```
 
+#### **Step 4:**  Drop the **SAMPLE_EVENTS_MOBILE_BASE** table
+
+```sql
+DROP TABLE TARGET_DB.ATOMIC.SAMPLE_EVENTS_MOBILE_BASE
+```
+You will now have the `ATOMIC.SAMPLE_EVENTS_MOBILE` created and loaded with sample data.
+
+{{% /tab %}}
+
+{{% tab name="BigQuery" %}}
+
+We have provided a public S3 bucket where we store the sample data in a csv format, you should be able to download it locally through this [link](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/Mobile/Mobile_sample_events ).
+
+You can load the sample data to the warehouse using your [Google Cloud console](https://console.cloud.google.com/) as described in the below steps.
+
+For more details please check out the official [BigQuery documentation](https://cloud.google.com/bigquery/docs/batch-loading-data#console).
+
+
+#### **Step 1:**  Create the ATOMIC schema
+If the ATOMIC schema doesn't exist, create it in your target database.
+
+```sql
+CREATE SCHEMA IF NOT EXISTS ATOMIC
+
+```
 ***
 
-#### **Step 5:**  Drop the **SAMPLE_EVENTS_BASE** table
+#### **Step 2:**  Upload the csv file
+
+2.1 Open the `BigQuery` page in the [Google Cloud console](https://console.cloud.google.com/).
+
+2.2 In the `Explorer` panel, expand your project and select the `atomic` schema/dataset. You should see the details panel open, if not, expand the `Actions` option (click on the three vertical dots) and click `Open`.
+
+2.3 In the `Details` panel, click `Create table`.
+
+3.4 On the `Create table` page fill out the sections:
+
```

Comment:
3.4?!?!?!?!?!?!?!?!?!?

Reply:
<img width="465" alt="image" src="https://github.com/snowplow/advanced-analytics-mobile-accelerator/assets/95634439/b1d03408-0de3-47ab-b967-b0067d8c9eae">

Reply:
Yeah 2.3 is right, it's the next number that's wrong

Reply:
The point was that it needs correcting in two more accelerators 😄